compiler/rustc_middle/src/ty/print/mod.rs

```diff
@@ -11,7 +11,7 @@ use tracing::{debug, instrument, trace};
 use crate::ty::{self, GenericArg, ShortInstance, Ty, TyCtxt};
 
 // `pretty` is a separate module only for organization.
-mod pretty;
+pub mod pretty;
```

Comment:
Why is this made pub? The contents of this mod are glob re-exported one line below, so this shouldn't be necessary.

Reply:
You are right! The compiler had suggested this and I did it without much thought.